src/devices/bus/qbus/pc11.h

```diff
@@ -0,0 +1,92 @@
+// license:BSD-3-Clause
+// copyright-holders:Sergey Svishchev
+/***************************************************************************
+
+	DEC PC11 paper tape reader and punch controller (punch not implemented)
+
+***************************************************************************/
+
+#pragma once
+
+#ifndef __PC11__
+#define __PC11__
+
+#include "emu.h"
+
+#include "qbus.h"
+
+#include "includes/pdp11.h"
```

Comment:
This is a circular dependency from src/devices on src/mame – that’s not acceptable.  This *needs* to be fixed.